src/rpc/handlers/Ledger.cpp

```diff
@@ -46,6 +46,18 @@ doLedger(Context const& context)
         diff = params.at("diff").as_bool();
     }
 
+    if (params.contains(JS(full)) && params.contains(JS(accounts)))
```

Comment:
Split this into two separate `if`s each checking its own field. If any of the fields is present (we don't care the type anymore) just return the `rpcNOT_SUPPORTED` code.